english/sing/continue/continue-to-sing.md

```diff
@@ -9,3 +9,9 @@ You must really work out that larynx.
 Do they call you the Iron Lung? Because they should.
 
 You start to feel something in your throat.
+
+You start to think about marshmallows, and s'mores,
+and that prompts you to sing [campfire songs](campfire_songs/rootbeer_song.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.